### ISSUE TITLE
Dynamic receiver quadrant positioning

### ISSUE THREAD

**Issue by zsmn** [MaracatronicsRobotics/Armorial-Suassuna-Legacy#33]
The receiver have 4 quadrants to be positioned: `QUADRANT_UP`, `QUADRANT_UPMID`, `QUADRANT_BOTMID` and `QUADRANT_BOT`.
The idea (generally) is to have one or two receivers activated with the attacker, so it's needed to know the best quadrant to position it (based in things like the quadrant that have more free angles or something like that), possibiliting the receivers to be at the best place to receive balls and make direct kicks to goal.

**Comment by zsmn** [comment]
Started at 530855e, for the next commit i'll change the heuristic to the **most free** quadrant, making an tiebreaker with the distance to it.

**Comment by zsmn** [comment]
I've maintained the 530855e heuristic (taking the most close quadrant to receiver, excluding the ball quadrant) at 694507d, because the receivers are swapping at quadrants too much faster... occasioning unsuported behaviours (goal invasions, crashes) and some things like that.
After a lot of tests, I saw that this simple heuristic isn't too bad, and with some adjusts at the receiver behaviour, it's making a lot of sense and with an higher goal rate (larger than the most free quadrant heuristic).
For future, i maintained the most free quadrant heuristic commented. So, if you want, undo this and try
